src/main/java/bio/terra/cli/businessobject/resource/BigQueryResolveOptions.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.cli.businessobject.resource;
+
+/** This enum specifies the possible ways to resolve a BigQuery dataset resource. */
+public enum BigQueryResolveOptions {
```

Comment:
nit: `BqResolveOptions` to be consistent with `BqDataset`, `BqTable` classes in the same package.